edx_lint/files/commitlint.config.js

```diff
@@ -6,9 +6,13 @@ module.exports = {
   rules: {
     'type-enum':
       [2, 'always', [
-        'revert', 'feat', 'fix', 'perf', 'docs', 'test', 'build', 'refactor', 'style', 'chore', 'temp',
+        'revert', 'feat', 'fix', 'perf', 'docs', 'test', 'build', 'refactor', 'style', 'chore', 'temp', 'squash',
       ]],
-
+    'type-enum':
```

Comment:
I don't _think_ duplicate keys are allowed.

Reply:
I also am skeptical. But this also shows the approach we'd probably need to take with your plugin idea. In other words, "squash" would need to be added to the accept list, and we'd need a custom rule that `never` allows a message that starts with "squash", but is just a warning.

Reply:
Yes, exactly, but since it's a dictionary it needs to have a different name; hence the need for a custom rule.

Reply:
Ah yes. I was mistakenly thinking it was up to their implementation internally whether or not they accepted the same key twice, but ignoring the fact that these rules are using an object and not a list, and this clearly won't work as you stated. And yes - a custom rule provides a new key - which is what we need.

Reply:
PR has been updated.